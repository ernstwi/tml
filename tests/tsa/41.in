init a

default location invar { x < 5 }

[b, c]
    invar { y < 10 }

[d, e]
    invar!

[f, g]
