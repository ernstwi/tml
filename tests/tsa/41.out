{
    "actions": [],
    "clocks": [
        "x",
        "y"
    ],
    "edges": [],
    "locations": [
        {
            "id": "a",
            "initial": true,
            "invariant": null
        },
        {
            "id": "b",
            "initial": false,
            "invariant": [
                {
                    "clock": "y",
                    "cmp": "lt",
                    "n": 10
                }
            ]
        },
        {
            "id": "c",
            "initial": false,
            "invariant": [
                {
                    "clock": "y",
                    "cmp": "lt",
                    "n": 10
                }
            ]
        },
        {
            "id": "d",
            "initial": false,
            "invariant": null
        },
        {
            "id": "e",
            "initial": false,
            "invariant": null
        },
        {
            "id": "f",
            "initial": false,
            "invariant": [
                {
                    "clock": "x",
                    "cmp": "lt",
                    "n": 5
                }
            ]
        },
        {
            "id": "g",
            "initial": false,
            "invariant": [
                {
                    "clock": "x",
                    "cmp": "lt",
                    "n": 5
                }
            ]
        }
    ]
}
